frontend/src/components/search/SearchSortBy.tsx

```diff
@@ -49,6 +49,7 @@ const SearchSortBy: React.FC<SearchSortByProps> = ({
         id="search-sort-by-select"
         onChange={handleChange}
         value={sortBy}
+        aria-label="Sort By"
```

Comment:
I wonder if we should use the [Truss USWDS Select component](https://github.com/trussworks/react-uswds/tree/main/src/components/forms/Select). 

```
    <Label htmlFor="search-sort-by show-for-sr" className="usa-sr-only">...</Label>
    <Select id="search-sort-by" name="search-sort-by-select">
      {options}
    </Select>
```

The `.usa-sr-only` class hides the label for sighted users.

Reply:
Opened a ticket to investigate separately https://github.com/HHS/simpler-grants-gov/issues/1637